### PR TITLE
Cleaned up submission document

### PR DESCRIPTION
**Pull request by dhairyya**
Added information about block
Cleared information about allow
Cleared misconception that allow skip filtering when it overrides

### CODE REVIEW

**Comment by dhairyya** [comment]
@chrisda  could you please approve this for me?

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ ms.date: 12/05/2022
 **Applies to**
 - [Exchange Online Protection](eop-about.md)
 - [Microsoft Defender for Office 365 plan 1 and plan 2](defender-for-office-365.md)
+- [Microsoft 365 Defender](../defender/microsoft-365-defender.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:30]:
```suggestion
- [Exchange Online Protection](../eop-about)
- [Microsoft Defender for Office 365 plan 1 and plan 2](../defender-for-office-365)
- [Microsoft 365 Defender](../defender/microsoft-365-defender)
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 2]
```diff
@@ -109,10 +110,11 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
 
 :::image type="content" source="../../media/admin-submission-email-block.png" alt-text="Submit a false negative (bad) email to Microsoft for analysis on the Submissions page in the Defender portal." lightbox="../../media/admin-submission-email-block.png":::
 
+After a few moments, the block entry will appear on the **Domains & addresses** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](tenant-allow-block-list-about.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:113]:
```suggestion
After a few moments, the block entry will appear on the **Domains & addresses** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](../tenant-allow-block-list-about).
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 3]
```diff
@@ -109,10 +110,11 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
...
+
 > [!NOTE]
-> For messages that were incorrectly blocked by [spoof intelligence](anti-spoofing-spoof-intelligence.md), a block entry for the domain pair is not created in the Tenant Allow/Block List.
 >
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 4]
```diff
@@ -109,10 +110,11 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
...
-> For messages that were incorrectly blocked by [domain or user impersonation protection](anti-phishing-policies-about.md#impersonation-settings-in-anti-phishing-policies-in-microsoft-defender-for-office-365), a block entry for the domain or sender is not created in the Tenant Allow/Block List. Instead, the domain or sender is added to the **Trusted senders and domains section** in the [anti-phishing policy](anti-phishing-policies-mdo-configure.md#use-the-microsoft-365-defender-portal-to-modify-anti-phishing-policies) that detected the message.
+> - For messages that were incorrectly blocked by [spoof intelligence](anti-spoofing-spoof-intelligence.md), a block entry for the domain pair is not created in the Tenant Allow/Block List.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
> For messages that were incorrectly blocked by [spoof intelligence](anti-spoofing-spoof-intelligence.md), a block entry for the domain pair is not created in the Tenant Allow/Block List.
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 5]
```diff
@@ -150,6 +152,8 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
 
 :::image type="content" source="../../media/admin-submission-file-block.png" alt-text="Submit a false negative (bad) email attachment to Microsoft for analysis on the Submissions page in the Defender portal." lightbox="../../media/admin-submission-file-block.png":::
 
+After a few moments, the block entry will appear on the **Files** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](tenant-allow-block-list-about.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:155]:
```suggestion
After a few moments, the block entry will appear on the **Files** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](../tenant-allow-block-list-about).
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 6]
```diff
@@ -186,29 +190,7 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
...
-   - **This file should have been categorized as**: Select **Malware** or **Unwanted Software**.
-
-   - **Choose the priority**: Select **Low - bulk file or file hash submission** or **Medium - standard submission** or **High - need immediate attention (3 allowed per org per day)**. If you're not sure, use your best judgment. This option is only available if you choose the option **Files** in **Select the submission type**.
-
-   - **Note for Microsoft**: Enter optional information in case there is anything else that needs to be added.
-
-   - Click on **Share feedback and relevant content with Microsoft**.
-
-   When you're finished, click **Submit**, and then click **Done**.
+After a few moments, the block entry will appear on the **URL** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](tenant-allow-block-list-about.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:193]:
```suggestion
After a few moments, the block entry will appear on the **URL** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](../tenant-allow-block-list-about).
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 7]
```diff
@@ -250,15 +232,18 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
 
    :::image type="content" source="../../media/admin-submission-email-allow.png" alt-text="Submit a false positive (good) email to Microsoft for analysis on the Submissions page in the Defender portal." lightbox="../../media/admin-submission-email-allow.png":::
 
-After a few moments, the allow entry will appear on the **Domains & addresses** or **Spoofed senders** tab on the **Tenant Allow/Block List** page.
+After a few moments, the allow entries will appear on the **Domains & addresses** or **Spoofed senders** or **URL** or **Files** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](tenant-allow-block-list-about.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:235]:
```suggestion
After a few moments, the allow entries will appear on the **Domains & addresses** or **Spoofed senders** or **URL** or **Files** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](../tenant-allow-block-list-about).
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 8]
```diff
@@ -250,15 +232,18 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
...
+> - For messages that were incorrectly blocked by [domain or user impersonation protection](anti-phishing-policies-about.md#impersonation-settings-in-anti-phishing-policies-in-microsoft-defender-for-office-365), the allow entry for the domain or sender is not created in the Tenant Allow/Block List. Instead, the domain or sender is added to the **Trusted senders and domains section** in the [anti-phishing policy](anti-phishing-policies-mdo-configure.md#use-the-microsoft-365-defender-portal-to-modify-anti-phishing-policies) that detected the message.
 > - During mail flow, if messages from the domain or email address pass other checks in the filtering stack, the messages will be delivered. For example, if [email authentication](email-authentication-about.md) passes, a message from a sender in the allow entry will be delivered.
+> - By default, allow entries for domains and email addresses, files and URLs are created for 30 days, while allow entries for spoofed senders never expire. 
+> - Microsoft will either learn from the allow entries for domains and email addresses, files and URLs within those 30 days, or automatically extend it for you.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
> - By default, allow entries for domains and email addresses, files, and URLs are created for 30 days, while allow entries for spoofed senders never expire. 
> - Microsoft will either learn from the allow entries for domains and email addresses, files, and URLs within those 30 days or automatically extend it for you.
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 9]
```diff
@@ -290,10 +275,15 @@ After a few moments, the allow entry will appear on the **Domains & addresses**
 
    :::image type="content" source="../../media/admin-submission-file-allow.png" alt-text="Submit a false positive (good) email attachment to Microsoft for analysis on the Submissions page in the Defender portal." lightbox="../../media/admin-submission-file-allow.png":::
 
-After a few moments, an allow entry will appear on the **Files** tab on the **Tenant Allow/Block List** page.
+After a few moments, the allow entry will appear on the **Files** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](tenant-allow-block-list-about.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:278]:
```suggestion
After a few moments, the allow entry will appear on the **Files** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](../tenant-allow-block-list-about).
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 10]
```diff
@@ -290,10 +275,15 @@ After a few moments, the allow entry will appear on the **Domains & addresses**
...
 
 > [!NOTE]
-> When the file is encountered again, it's not sent for [Safe Attachments](safe-attachments-about.md) detonation or file reputation checks, and all other file-based filters are skipped. During mail flow, if messages containing the file pass other non-file checks in the filtering stack, the messages will be delivered.
+> 
+> - By default, allow entries for domains and email addresses, files and URLs are created for 30 days, while allow entries for spoofed senders never expire. 
+> - Microsoft will either learn from the allow entries for domains and email addresses, files and URLs within those 30 days, or automatically extend it for you.
+> - When the file is encountered again during mailflow, [Safe Attachments](safe-attachments-about.md) detonation or file reputation checks, and all other file-based filters are overriden. If all the other entities in the email are found clean by the filtering system, the email will be delivered. 
+> - During time of click, all file-based filters including [Safe Attachments](safe-attachments-about.md) detonation or file reputation checks are overriden, allowing end user to access the file.
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
> - By default, allow entries for domains and email addresses, files, and URLs are created for 30 days, while allow entries for spoofed senders never expire. 
> - Microsoft will either learn from the allow entries for domains and email addresses, files, and URLs within those 30 days or automatically extend it for you.
> - When the file is encountered again during mailflow, [Safe Attachments](safe-attachments-about.md) detonation or file reputation checks and all other file-based filters are overridden. If all the other entities in the email are found clean by the filtering system, the email will be delivered. 
> - During selection, all file-based filters, including [Safe Attachments](safe-attachments-about.md) detonation or file reputation checks are overridden, allowing the end user to access the file.
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 11]
```diff
@@ -325,36 +315,14 @@ After a few moments, an allow entry will appear on the **Files** tab on the **Te
 
    :::image type="content" source="../../media/admin-submission-url-allow.png" alt-text="Submit a false positive (good) URL to Microsoft for analysis on the Submissions page in the Defender portal." lightbox="../../media/admin-submission-url-allow.png":::
 
-After a few moments, an allow entry will appear on the **URL** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](tenant-allow-block-list-about.md).
+After a few moments, the allow entry will appear on the **URL** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](tenant-allow-block-list-about.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:318]:
```suggestion
After a few moments, the allow entry will appear on the **URL** tab on the **Tenant Allow/Block List** page. For more information about the Tenant Allow/Block List, see [Manage your allows and blocks in the Tenant Allow/Block List](../tenant-allow-block-list-about).
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 12]
```diff
@@ -325,36 +315,14 @@ After a few moments, an allow entry will appear on the **Files** tab on the **Te
...
-
-   - **Note for Microsoft**: Enter optional information in case there is anything else that needs to be added.
-
-   - Click on **Share feedback and relevant content with Microsoft**.
-
-   When you're finished, click **Submit**, and then click **Done**.
+> - By default, allow entries for domains and email addresses, files and URLs are created for 30 days, while allow entries for spoofed senders never expire. 
+> - Microsoft will either learn from the allow entries for domains and email addresses, files and URLs within those 30 days, or automatically extend it for you.
+> - When the URL is encounterd again during mailflow, [Safe Links](safe-links-about.md) detonation or URL reputation checks, and all other URL-based filters are overriden. If all the other entities in the email are found clean by the filtering system, the email will be delivered. 
+> - During time of click, all URL-based filters including [Safe Links](safe-links-about.md) detonation or URL reputation checks are overriden, allow end user user to access content hosted by the URL.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
> - By default, allow entries for domains and email addresses, files, and URLs are created for 30 days, while allow entries for spoofed senders never expire. 
> - Microsoft will either learn from the allow entries for domains and email addresses, files, and URLs within those 30 days or automatically extend it for you.
> - When the URL is encountered again during mailflow, [Safe Links](safe-links-about.md) detonation or URL reputation checks, and all other URL-based filters are overridden. If all the other entities in the email are found clean by the filtering system, the email will be delivered. 
> - During selection, all URL-based filters, including [Safe Links](safe-links-about.md) detonation or URL reputation checks are overridden, allowing the end user to access content hosted by the URL.
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 13]
```diff
@@ -250,15 +232,18 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
...
 > - Allows are added during mail flow, based on which filters determined the message to be malicious. For example, if the sender and a URL in the message were determined to be bad, an allow entry is created for the sender, and an allow entry is created for the URL.
 > - When that entity (domain or email address, URL, file) is encountered again, all filters associated with that entity are skipped. For an email, all other entities are still evaluated by the filtering system before making a decision.
+> - For messages that were incorrectly blocked by [domain or user impersonation protection](anti-phishing-policies-about.md#impersonation-settings-in-anti-phishing-policies-in-microsoft-defender-for-office-365), the allow entry for the domain or sender is not created in the Tenant Allow/Block List. Instead, the domain or sender is added to the **Trusted senders and domains section** in the [anti-phishing policy](anti-phishing-policies-mdo-configure.md#use-the-microsoft-365-defender-portal-to-modify-anti-phishing-policies) that detected the message.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:243]:
```suggestion
> - For messages that were incorrectly blocked by [domain or user impersonation protection](../anti-phishing-policies-about#impersonation-settings-in-anti-phishing-policies-in-microsoft-defender-for-office-365), the allow entry for the domain or sender is not created in the Tenant Allow/Block List. Instead, the domain or sender is added to the **Trusted senders and domains section** in the [anti-phishing policy](../anti-phishing-policies-mdo-configure#use-the-microsoft-365-defender-portal-to-modify-anti-phishing-policies) that detected the message.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the suggested changes. Many thanks.

**Pull request merged**